conan/tools/system/pip_manager.py

```diff
@@ -0,0 +1,45 @@
+import venv
+import platform
+import os
+
+from conan.errors import ConanException
+from conan.tools.env.environment import Environment
+
+
+class PipEnv:
+    accepted_install_codes = [0]
+    accepted_check_codes = [0, 1]
+
+    def __init__(self, conanfile, folder=None):
+        self._conanfile = conanfile
+        self.env_name = f"pip_venv_{self._conanfile.name}"
+        self._env_dir = os.path.abspath(os.path.join(folder or self._conanfile.build_folder, self.env_name))
+        self.bin_dir = os.path.join(self._env_dir, "Scripts" if platform.system() == "Windows" else "bin")
+        self._python_exe = os.path.join(self.bin_dir, "python.exe" if platform.system() == "Windows" else "python")
+
+    def _conanfile_run(self, command, accepted_returns):
+        ret = self._conanfile.run(command, ignore_errors=True, quiet=True)
+        if ret not in accepted_returns:
+            raise ConanException("Command '%s' failed" % command)
+        return ret
```

Comment:
I think we don't need this any more?

Reply:
I mean, that I guess it was only used because of the `accepted_returns` but you are not accepting non-error codes any more, so I think we should just use `self._conanfile.run` instead of the wrapper with `accepted_returns`